learning/courses/basics-of-quantum-information/entanglement-in-action/qiskit-implementation.ipynb

```diff
@@ -0,0 +1,1588 @@
+    "\n",
+    "The way a nonlocal game works is that the referee first asks each of Alice and Bob a question.\n",
+    "We'll use the letter $x$ to refer to Alice's question and $y$ to refer to Bob's question.\n",
+    "Here we're thinking of $x$ and $y$ as being classical states, and in the CHSH game $x$ and $y$ are bits.\n",
+    "\n",
+    "The referee uses *randomness* to select these questions.\n",
+    "To be precise, there is some probability $p(x,y)$ associated with each possible pair $(x,y)$ of questions, and the referee has vowed to choose the questions randomly, at the time of the game, in this way.\n",
+    "Everyone, including Alice and Bob, knows these probabilities — but nobody knows specifically which pair $(x,y)$ will be chosen until the game begins.\n",
+    "\n",
+    "After Alice and Bob receive their questions, they must then provide answers: Alice's answer is $a$ and Bob's answer is $b.$\n",
+    "Again, these are classical states in general, and bits in the CHSH game.\n",
+    "\n",
+    "At this point the referee makes a decision: Alice and Bob either *win* or *lose* depending on whether or not the pair of answers $(a,b)$ is deemed correct for the pair of questions $(x,y)$ according to some fixed set of rules.\n",
+    "Different rules mean different games, and the rules for the CHSH game specifically are described in the section following this one.\n",
+    "As was already suggested, the rules are known to everyone.\n",
+    "\n",
+    "The following diagram provides a graphic representation of the interactions.\n",
+    "\n",
+    "![Nonlocal game](/learning/images/courses/basics-of-quantum-information/entanglement-in-action/qiskit-implementation/nonlocal-game.avif)\n",
+    "\n",
+    "It is the uncertainty about which questions will be asked, and specifically the fact that each player doesn't know the other player's question, that makes nonlocal games challenging for Alice and Bob — just like colluding suspects in different rooms trying to keep their story straight.\n",
+    "\n",
+    "A precise description of the referee defines an instance of a nonlocal game.\n",
+    "This includes a specification of the probabilities $p(x,y)$ for each question pair along with the rules\n",
+    "that determine whether each pair of answers $(a,b)$ wins or loses for each possible question pair $(x,y).$\n",
+    "\n",
+    "We'll take a look at the CHSH game momentarily, but before that let us briefly acknowledge that it's also interesting to consider other nonlocal games.\n",
+    "In fact, it's extremely interesting; there are some pretty simple nonlocal games for which it's currently not known how well Alice and Bob can play using entanglement.\n",
+    "The set-up is simple, but there's complexity at work — and for some games it can be impossibly difficult to compute best or near-best strategies for Alice and Bob.\n",
+    "This is the mind-blowing nature of the non-local games model.\n",
```

Comment:
```suggestion
    "This is the surprisingly unintuitive nature of the nonlocal games model.\n",
```
Not sure if this construes the strength of the original "mind-blowing", but am trying to come up with something that might be translation-friendlier, and that's the best I could do :)